build.gradle.kts

```diff
@@ -137,9 +136,7 @@ dependencies {
 
   // Align versions of all Kotlin components
   implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
-
-  // Use the Kotlin JDK 8 standard library.
-  implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4")
```

Comment:
Could you please remove this and test again?
Apparently, it has no effect for the moment, so there's no need to keep it ;)